### DISCUSSION TITLE
New familly of analysis function (ttc, pdf, polarisation,...)

### DISCUSSION THREAD

**Discussion by ThoChat** [PedestrianDynamics/PedPy#318] [General]
What should be the name of the group of methods which can be used tho characterise different types of crowd? Such as the Pair-Distribution function, the overall Time-To-Collision, the crowd polarisation and other dimensionless numbers.
Do you have any opinion?

**Comment by chraibi** [comment]
Characterization methods or Spatial-Temporal Analysis Methods.

**Reply by chraibi** [reply]
@ThoChat I opened a PR to address this. See #331
